rpc/common/moc_common_common.proto

```diff
@@ -160,3 +160,9 @@ enum ImageSource {
         HTTP_SOURCE  = 2;
         CLONE_SOURCE = 3;
 }
+
+enum HyperVGeneration {
+	HyperVGenerationV1 = 0;
```

Comment:
Should we flip this to keep V2 as default

Reply:
in moc-sdk and wssd-sdk I make sure the default is V2. https://github.com/microsoft/moc-sdk-for-go/pull/96/files and https://github.com/microsoft/wssd-sdk-for-go/pull/187/files